JBBCode/CodeDefinition.php

```diff
@@ -11,28 +11,28 @@
  */
 class CodeDefinition
 {
-    /* NOTE: THIS PROPERTY SHOULD ALWAYS BE LOWERCASE; USE setTagName() TO ENSURE THIS */
+    /* @var string NOTE: THIS PROPERTY SHOULD ALWAYS BE LOWERCASE; USE setTagName() TO ENSURE THIS */
```

Comment:
Nope, that's the inline style recognized by most (all?) IDEs. If you're experiencing any troubles, I'll gladly "fix" it, though.